components/article-footer/server.js

```diff
@@ -36,11 +36,9 @@ export class ArticleFooter extends ServerComponent {
  */
 function Contribute(context) {
   return html`<a
-    class="article-footer__contribute external"
-    href="https://github.com/mdn/content/blob/main/CONTRIBUTING.md"
-    title=${context.l10n`This will take you to our contribution guidelines on GitHub.`}
+    class="article-footer__contribute"
+    href=${`/${context.locale}/docs/MDN/Community/Getting_started`}
     target="_blank"
```

Comment:
Does this still need to open in a new tab? I don't think we open internal links in a new tab anywhere else.

```suggestion
```

Reply:
> Does this still need to open in a new tab?

nope! Let's get rid of it